recipes/mpi4py-fft/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set build = 0 %}
+{% set name = "mpi4py-fft" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/mpi4py/mpi4py-fft/get/{{ version }}.tar.bz2
+  sha256: d697e836d436c12da721b4bcc984f3cc99fff6e72b4a9e3a31da93bc7a89db40
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  string: py{{ py }}_{{ mpi }}_{{ build }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - {{ mpi }}
+    - mpi4py
+    - numpy
+    - nomkl
```

Comment:
I suspect `nomkl` isn't needed here, that's more of a user choice. Does mpi4py directly link blas/lapack? If so, perhaps `openblas` belongs here instead, otherwise I think this can just be removed.

Reply:
Thanks for looking at this @minrk :-) But we need nomkl because of the fft-libraries. It is really important, because mkl is under the hood setting some FFTW-parameters that makes everything else break. I have not dug very deep, but removing mkl from the equation at least solves the problem.

Reply:
When building with conda-forge packages, openblas is used in general, so mkl ought to not be pulled in. I'm okay with this as it is for now, but we might want to investigate further once the feedstock is created.